nodeadm/internal/feature-gates/feature-gates.go

```diff
@@ -0,0 +1,18 @@
+package featuregates
+
+type FeatureGate = string
+
+const (
+	// InstanceIdNamedNode When this flag is enabled, override the hostname with instanceId
+	InstanceIdNamedNode = FeatureGate("instanceIdNamedNode")
```

Comment:
nit: are feature gate keys usually lowercase or uppercase? kubelet's are uppercase

Reply:
ack, updating it